aldryn_jobs/models.py

```diff
@@ -241,10 +239,7 @@ def get_absolute_url(self, language=None):
         category_slug = self.category.safe_translation_getter(
             'slug', language_code=language
         )
-        if self.app_config_id:
-            namespace = self.app_config.namespace
-        else:
-            namespace = 'aldryn_jobs'
+        namespace = getattr(self.category.app_config, "namespace", "aldryn_jobs")
```

Comment:
under what circumstance would this throw an ```AttributeError```? would there be a case where category does not have app_config?